pyEDAA/Launcher/__init__.py

```diff
@@ -67,22 +71,27 @@ def get_version(file_path):
 	return str(version_major + "." + version_minor)
 
 
+@export
 def get_vivado_versions(install_path):
-	return [directory.name for directory in install_path.iterdir()]
+	return [item.name for item in install_path.iterdir() if item.is_dir()]
```

Comment:
@stefanunrein I would suggest to change this list comprehension to a generator. This is more efficient in your case.